test/library/gui/widgets.Label.lua

```diff
@@ -0,0 +1,117 @@
+-- test -dhack/scripts/devel/tests -twidgets.Label
+
+--reload('gui')
+--reload('gui.widgets')
```

Comment:
this should not be necessary. the test harness loads all lua libraries cleanly before starting the tests.